content/blog/TinaCMS-Version-21.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: TinaCMS Version 2.1
+date: '2024-07-24T14:00:00.000Z'
+last_edited: '2024-07-24T14:00:00.000Z'
+author: Jack Pettit
+---
+
+We are excited to announce the release of `tinacms@2.1.0` and `@tinacms/cli@1.5.51`!
+
+### What's New:
+
+1. **Removed Deprecated Hook**
+   * A deprecated hook, which was part of legacy internal systems, has now been fully removed. This change does not affect the commonly used `useTina` hook.
+2. **Next.js Version Upgrade**
+   * We've upgraded the Next.js version in TinaCMS packages from 12 to 14, along with several other dependency updates to ensure compatibility and improved performance.
+3. **PNPM Support for `create-tina-app`**
+   * You can now use PNPM as an option when creating a Tina app, offering more flexibility and efficiency in your package management.
+4. **React 18.3.1 Integration**
+   * TinaCMS has been upgraded to use React 18.3.1, bringing you the latest features and improvements from React.
+5. **New `toolbarOverride` Option for Rich Text Fields**
+   * A new `toolbarOverride` option has been added to rich text fields, giving you greater control over your editor toolbar configurations. Checkout the [docs](/docs/reference/types/rich-text/) for more info
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1721881770/blog-media/2.1/ray-so-export_3_x7baul.png)
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1721881650/blog-media/2.1/SCR-20240725-mvmu_gfqxxv.png)
+
+6. **Enhanced Media Uploads with Cloudinary Support**
+   * The default media upload types now include support for Cloudinary files, allowing you to upload videos and other media directly through the media manager.
+
+![](https://res.cloudinary.com/forestry-demo/image/upload/v1721880641/blog-media/2.1/351543729-f4b352d6-83c9-4f13-af38-8faa0e41f993_q8kuz7.png)
+
+7. **Mobile Resolution Support**
+   * TinaCMS now supports various screen sizes and mobile devices, including iOS and Android, providing a seamless editing experience across all your devices.
+
+![](/img/blog/tinacms-2-1-mobile.png)
```

Comment:
I can't seem to preview this image nor open the url :(

![image](https://github.com/user-attachments/assets/f00ed37a-27e6-4a81-9921-0729af0cf126)
**Figure: Image not appearing**

Reply:
Looks like its just my GitHub